Aquality.Selenium.Core/src/Aquality.Selenium.Core/Elements/ElementFinder.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Aquality.Selenium.Core.Elements.Interfaces;
+using Aquality.Selenium.Core.Localization;
+using Aquality.Selenium.Core.Logging;
+using Aquality.Selenium.Core.Waitings;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace Aquality.Selenium.Core.Elements
+{
+    /// <summary>
+    /// Implementation of <see cref="IElementFinder"/>
+    /// </summary>
+    public class ElementFinder : IElementFinder
+    {
+        public ElementFinder(Logger logger, LocalizationLogger localizationLogger, LocalizationManager localizationManager, ConditionalWait conditionalWait)
```

Comment:
do not we have to use interfaces everywhere?

Reply:
that's the question :)